doc/cookbook/source/examples/clustering/gmm.rst

```diff
@@ -0,0 +1,47 @@
+=======================
+Gaussian Mixture Models
+=======================
+
+A Gaussian Mixture Model is a probabilistic model that assumes all the data points are generated from a mixture of a finite number of Gaussian distributions with unknown parameters.
+
+.. math::
+
+    p(\theta|\boldsymbol{X}) = \sum_{i=1}^{K}{\phi_i \cal{N}(\boldsymbol{\mu_i}, \boldsymbol{\Sigma_i})}
+
+where :math:`p(\theta|\boldsymbol{X})` is posterior distribution, :math:`K` denotes number of mixture components, :math:`\phi_i` denotes weight for :math:`i^{th}` component, :math:`\cal{N}` denotes normal distribution with mean vector :math:`\boldsymbol{\mu_i}` and covariance matrix :math:`\boldsymbol{\Sigma_i}`.
+
+See Chapter 20 in :cite:`barber2012bayesian` for a detailed introduction.
+
```

Comment:
Mention that this implementation uses the EM algorithm which is executed until some convergence criterion (which) is met